src/components/task/index.cjsx

```diff
@@ -23,7 +23,7 @@ module.exports = React.createClass
   propTypes:
     id: React.PropTypes.string
 
-  displayName: 'ReadingTask'
+  displayName: 'Task'
 
```

Comment:
The component was incorrectly named here.   This is the main component that wraps the steps.